scipy/stats/tests/test_multicomp.py

```diff
@@ -250,6 +250,61 @@ def test_alternatives(self, alternative, statistic, pvalue):
             )
             assert_allclose(res.pvalue, pvalue, atol=1e-7)
 
+    @pytest.mark.parametrize(
+        'alternative, allowance, ci_low, ci_high',
+        [
+            ('two-sided', 11, [0, -9, -16], [22, 13, 6]),
+            ('less', 9, [2, -7, -14], [np.inf, np.inf, np.inf]),
+            ('greater', 9, [-np.inf, -np.inf, -np.inf], [20, 11, 4])
+        ]
+    )
+    def test_allowance(self, alternative, allowance, ci_low, ci_high):
+        # Example (a) from Dunnett1995
+        rng = np.random.default_rng(189117774084579816190295271136455278291)
+        samples = [
+            [55, 64, 64],
+            [55, 49, 52],
+            [50, 44, 41]
+        ]
+        control = [55, 47, 48]
+
+        res = stats.dunnett(
+            *samples, control=control, alternative=alternative,
+            random_state=rng
+        )
+        allowance_ = res._allowance(confidence_level=0.95)
+        assert_allclose(allowance_, allowance, atol=1)
+
+        assert res._ci is None
+        assert res._ci_cl is None
+        ci = res.confidence_interval(confidence_level=0.95)
+        assert_allclose(ci.low, ci_low, atol=1)
+        assert_allclose(ci.high, ci_high, atol=1)
```

Comment:
I feel like these are pretty loose. Can you run these with R's [`multcomp`](https://cran.r-project.org/web/packages/multcomp/index.html) and get more precise values to compare against?